src/main/java/utils/FormUrlEncodedParsingStrategy.java

```diff
@@ -0,0 +1,28 @@
+package utils;
+
+import static constant.Constant.*;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class FormUrlEncodedParsingStrategy implements HttpBodyParsingStrategy {
+    @Override
+    public Map<String, String> parse(String body) {
+        if (body.isBlank()) {
+            return Map.of();
+        }
+
+        Map<String, String> map;
+        try {
+            map = Arrays.stream(body.split(PARAMETER_SEPARATOR))
+                    .map(s -> s.split(PARAMETER_EQUAL_SIGN))
+                    .map(arr -> Map.entry(arr[0], arr[1]))
+                    .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
```

Comment:
`Map.Entry`로 변경할 필요 없이, 아래와 같이 바로 변경하면 될 것 같아요. 🙂
```suggestion
                    .collect(Collectors.toMap(arr -> arr[0], arr -> arr[1]));
```